js/main.js

```diff
@@ -0,0 +1,39 @@
+let string = 'some test string';
+
+alert(string[0]); //получаем первую букву
+alert(string.length); //определяем длину
+alert(string.length-1); //последний символ строки
+alert(string[15]); //получаем последнюю буквы
```

Comment:
`string[string.length - 1]` - cause you shouldn't count index number by yourself